install/linux/usr/share/odemis/sim/fastem-sim.yaml

```diff
@@ -0,0 +1,64 @@
+# Minimal configuration file to start the Fast-EM GUI
+FASTEM-sim: {
+    class: Microscope,
+    role: mbsem,
+    children: ["SEM", "EBeam Scanner", "EBeam Focus", "SED",
+               "Sample Stage"],
+}
+
+"SEM": {
+    class: simsem.SimSEM,
+    # TODO remove role when children components are implemented (chamber camera, chamber and sed)
+    role: sem,
```

Comment:
You can remove the role (ie, set it to "null"). And also remove the "SEM" from the Microscope children.